apps/pie-docs/src/patterns/cookie-banner/code/code.md

```diff
@@ -111,12 +102,12 @@ Use of locales in Vue:
 // Vue templates (using Nuxt 3)
 <script setup lang="ts">
   import '@justeattakeaway/pie-cookie-banner';
-  import enGB from "@justeattakeaway/pie-cookie-banner/locales/en-gb.json";
 </script>
 
 <pie-cookie-banner
   hasPrimaryActionsOnly
-  :locale='enGB'
+  :language='en'
+  :country='GB'
```

Comment:
Actually I think this property is case-sensitive, so we should probably stick to lowercase, or handle uppercase properly.

E.g., on storybook if you manually set the language to `ca` and the country to `ES` it shows English even though `ca-es` is a supported pairing.